requirements/developer.txt

```diff
@@ -7,7 +7,7 @@ pytest-cov==4.1.0
 grpcio==1.54.2
 protobuf==4.23.1
 grpcio-tools==1.54.2
-transformers==4.30.0
+transformers>=4.34.0
```

Comment:
found 4.34.o can not work with optimum. could you change to >=4.31.0?

Reply:
I tried it on CPU .. seems to work

```
>>> from optimum.bettertransformer import BetterTransformer
>>> import transformers
>>> transformers.__version__
'4.34.0'
>>> from optimum.bettertransformer import BetterTransformer
>>> 

```

Reply:
Works on GPU too